server/update_test.go

```diff
@@ -51,7 +51,7 @@ func TestReachServer(t *testing.T) {
 		t.Fatal("Expecting some URL.")
 	}
 
-	res, err = param.CheckForUpdate(fmt.Sprintf("http://%s/update/getlantern/lantern", localAddr), up)
+	res, err = param.CheckForUpdate(fmt.Sprintf("http://%s/update/lantern", localAddr), up)
```

Comment:
This was an error inherited from https://github.com/getlantern/autoupdate-server/pull/14